src/roulette.py

```diff
@@ -126,3 +126,172 @@ def get(self, bin: int) -> Bin:
+        split_bet_odds = 17
+        vertical_split_bet_numbers = {_ for _ in range(1, 34)}
+        for number in vertical_split_bet_numbers:
+            vertical_split_bet = Outcome(str(number) + "-" + str(number + 3), split_bet_odds)
+            wheel.addOutcome(number, vertical_split_bet)
+            wheel.addOutcome(number + 3, vertical_split_bet)
+
+    @staticmethod
+    def create_street_bets(wheel: Wheel) -> None:
+        street_bet_odds = 11
+        street_bet_numbers = set(number * 3 + 1 for number in range(12))
+        for number in street_bet_numbers:
+            street_bet = Outcome(str(number) + "-" + str(number + 1) + "-" + str(number + 2),
+                                 street_bet_odds)
+            wheel.addOutcome(number, street_bet)
+            wheel.addOutcome(number + 1, street_bet)
+            wheel.addOutcome(number + 2, street_bet)
+
+    @staticmethod
+    def create_corner_bets(wheel: Wheel) -> None:
+        odds_for_corner_bet = 8
+
+        corner_bet_numbers = {number * 3 + 1 for number in range(11)} | {number * 3 + 2 for
+                                                                         number in range(11)}
+        for number in corner_bet_numbers:
+            first_column_corner_bet = Outcome(str(number) + "-" + str(number + 1) + "-" +
+                                              str(number + 3) + str(number + 4),
+                                              odds_for_corner_bet)
+            wheel.addOutcome(number, first_column_corner_bet)
+            wheel.addOutcome(number + 1, first_column_corner_bet)
+            wheel.addOutcome(number + 3, first_column_corner_bet)
+            wheel.addOutcome(number + 4, first_column_corner_bet)
+
+    @staticmethod
+    def create_line_bets(wheel: Wheel) -> None:
+        line_bet_odds = 5
+        line_bet_numbers = {number * 3 + 1 for number in range(11)}
+        for number in line_bet_numbers:
+            line_bet = Outcome(str(number) + "-" + str(number + 1) + "-" + str(number + 2) + "-" +
+                               str(number + 3) + "-" + str(number + 4) + "-" + str(number + 5),
+                               line_bet_odds)
+            wheel.addOutcome(number, line_bet)
+            wheel.addOutcome(number + 1, line_bet)
+            wheel.addOutcome(number + 2, line_bet)
+            wheel.addOutcome(number + 3, line_bet)
+            wheel.addOutcome(number + 4, line_bet)
+            wheel.addOutcome(number + 5, line_bet)
+
+    @staticmethod
+    def generate_five_bets(wheel: Wheel) -> None:
+        odds_for_five_bet = 6
+        five_bet_name = "00-0-1-2-3"
+        five_bet = Outcome(five_bet_name, odds_for_five_bet)
+        bin_indexes_for_five_bet = {0, 1, 2, 3, 37}
+
+        for index in bin_indexes_for_five_bet:
+            wheel.addOutcome(index, five_bet)
+
+    @staticmethod
+    def create_even_money_bets(wheel: Wheel) -> None:
+        even_bet_odds = 1
+        red_bet_name = "Red"
```

Comment:
a blank line odds and bet names will improve readability.

Reply:
Done. Thanks.